index.html

```diff
@@ -13,23 +13,202 @@
+					<li><a id="language-name" href="#">html</a></li>
+					<li><a id="language-name" href="#">css</a></li>
+					<li><a id="language-name" href="#">javascript</a></li>
+				</ul>
+				<div><a class="see-projects" href="#">See Projects</a></div>
+			</main>
+		</div>
+
+		<div class="projects">
+			<div class="project-images">
+				<img src="./images/tonic2.svg" alt="Picture of project">
+			</div>
+			<main class="project-title">
+				<h2>Tonic</h2>
+				<ul class="list">
+					<li id="canopy">Canopy</li>
+					<li><img src="./images/Counter.svg" alt=""></li>
+					<li id="backend">Back End Dev</li>
+					<li><img src="./images/Counter.svg" alt=""></li>
+					<li id="backend">2015</li>
+				</ul>
+				<p class="description">A daily selection of privately personalized reads; no accounts or sign-ups required.
+				</p>
+				<ul class="program-languages">
+					<li><a id="language-name" href="#">html</a></li>
+					<li><a id="language-name" href="#">css</a></li>
+					<li><a id="language-name" href="#">javascript</a></li>
+				</ul>
+				<div><a class="see-projects" href="#">See Projects</a></div>
+			</main>
+		</div>
+
+		<div class="projects">
+			<div class="project-images">
+				<img src="./images/multipost.svg" alt="Picture of project">
+			</div>
+			<main class="project-title">
+				<h2>Multi-Post<br>Stories </h2>
+				<ul class="list">
+					<li id="canopy">Canopy</li>
+					<li><img src="./images/Counter.svg" alt=""></li>
+					<li id="backend">Back End Dev</li>
+					<li><img src="./images/Counter.svg" alt=""></li>
+					<li id="backend">2015</li>
+				</ul>
+				<p class="description">A daily selection of privately personalized reads; no accounts or sign-ups required.
+				</p>
+				<ul class="program-languages">
+					<li><a id="language-name" href="#">html</a></li>
+					<li><a id="language-name" href="#">css</a></li>
+					<li><a id="language-name" href="#">javascript</a></li>
+				</ul>
+				<div><a class="see-projects" href="#">See Projects</a></div>
+			</main>
+		</div>
+
+		<div class="about-container">
+			<h2 class="about-heading">About<br>Myself</h2>
+			<main>
+				<p id="intro">Hello I’m a software developer! I can help you build a product , feature or
+					website Look through
+					some of my
+					work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+				</p>
+				<p id="lets-connect">LET'S CONNECT</p>
+				<ul class="socials about-socials">
+					<li><a href="#"><img src="./images/github.svg" alt="Github logo"></a></li>
+					<li><a href="#"><img src="./images/linkedin.svg" alt="Linkedin Logo"></a></li>
+					<li><a href="#"><img src="./images/angellist.svg" alt="Angellist Logo"></a></li>
+					<li><a href="#"><img src="./images/twitter.svg" alt="Twitter Logo"></a></li>
+					<li><a href="#"><img src="./images/medium.svg" alt="Medium Logo"></a></li>
+				</ul>
+				<div><a class="see-projects resume" href="#">Get my resume</a></div>
```

Comment:
- [ ] Good job so far :+1:  But there are some differences between your design and the Figma template :cry: Please try to make your design fit mobile screen size `375`px. and ensure you that stick to the Figma template as much as possible :smile: 

#### Your design 
![Screenshot from 2022-09-28 10-48-31](https://user-images.githubusercontent.com/6615394/192764062-9413c4e6-2e4b-4f52-9eb1-cd432916c8e7.jpg)

#### Figma
![Screenshot from 2022-09-28 10-51-15](https://user-images.githubusercontent.com/6615394/192764066-9e8fd752-7840-4726-b2ef-ff99d9bf98a4.png)